itests/src/test/resources/com/nextdoor/bender/handler/config_kinesis.yaml

```diff
@@ -20,6 +20,8 @@ sources:
         key: metadata
         excludes:
           - sourceLagMs
+  - type: DeleteFieldOperation
+    key_field: "$.metadata"
```

Comment:
I would delete a different field or have a different test file because part of this test also tests the make sure `metadata` is correct.

Reply:
hmm let me think about how to structure this, may refactor some stuff to make this easier to add tests.

Reply:
Restructured it, let me know what you think.